openedx/features/edly/api/v1/views/user_sites.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from lms.djangoapps.mobile_api.decorators import mobile_view
+from openedx.features.edly.api.serializers import UserSiteSerializer
+
+
+@mobile_view()
+class UserSitesViewSet(viewsets.ViewSet):
+    permission_classes = (IsAuthenticated,)
+    serializer = UserSiteSerializer
```

Comment:
We need to explicitly whitelist those HTTP methods which we allow, e.g. `GET`

Reply:
Right now we only implemented `list` method so only a GET request is allowed all other requests would get `Method not allowed` 
<img width="889" alt="Screenshot 2020-10-21 at 11 42 49 AM" src="https://user-images.githubusercontent.com/15142776/96682982-ff0b0f80-1392-11eb-91b8-f48bd93483a8.png">
<img width="889" alt="Screenshot 2020-10-21 at 11 42 55 AM" src="https://user-images.githubusercontent.com/15142776/96683002-02060000-1393-11eb-97a7-a32ac8157c3f.png">

Reply:
@iam-mhaseeb is this method restriction enforced by the edX decorator `@mobile_view` ?

Reply:
@zubair-arbi Whole viewset is restricted under mobile_view edX decorator.

Reply:
Done